ignore-symbol-check.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 447871b..55ed691 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -225,7 +225,7 @@ if(APPLE)
+ endif()
+
+ set(CMAKE_REQUIRED_LIBRARIES OBS::libobs)
+-check_symbol_exists(obs_properties_add_color_alpha "obs-module.h" HAVE_OBS_PROP_ALPHA)
++set(HAVE_OBS_PROP_ALPHA, ON)
```

Comment:
The symbol check relies on the wrong header file, a fix should be upstreamed to the plugin.

```suggestion
-check_symbol_exists(obs_properties_add_color_alpha "obs-module.h" HAVE_OBS_PROP_ALPHA)
+check_symbol_exists(obs_properties_add_color_alpha "obs-properties.h" HAVE_OBS_PROP_ALPHA)
```

Reply:
I tried this at first but it still fails compiling with the same error.

Reply:
Yeah this is a CMake issue, I'll fix it upstream.